### ISSUE TITLE
It activate accents while it shouldn't

### ISSUE THREAD

**Issue by Mahfoudh94** [damienleroy/PowerAccent#80]
I am using PowerAccent alongside many PowerToys tools, I have win+shift+z and win+shift+x to switch between desktops.
since this tool was added (and I use) when I do my shortcut it doesn't work because it catches the accents for z or x, it is probable because the 2 shortcuts maps to ctrl+win+left/right and my accecnt key is set to left/right arrow, I tried changing the key to space and it works but the space bar isn't convinient to me, I tried rising the delay to 2s but  neither accent nor shortcut worked.
maybe the tool should get more shortcut keys or "type then switch" feature or maybe accent list to show all available accents.

**Comment by damienleroy** [comment]
Sorry I don't really understand, is that mean you are using a key mapper to use win+shift+z/x to switch between desktops?
On my side I am often using ctrl+win+left/right and it works well.
>and my accecnt key is set to left/right arrow
Same question there. PowerAccent is also using arrows to navigate in the selector. Are you using it for an another usage?

Therefore when PA catch the key, it shouldn't stop the action related to this key with an another program/windows feature.
Except for a combination like `letter+space` or `letter+arrow left/right`.

**Comment by Mahfoudh94** [comment]
yes as you say: I map win+shift+z => ctrl+win+arrowLeft and win+shift+x => ctrl+win+arrowRight
the problem is that power accent catches the arrow key after the mapping and stops the operation

**Comment by damienleroy** [comment]
Can you please tell me which is the software you are using to remap keys?
(I tryed with Keyboard Manager from PowerToys but it is limited to 2 keys combination at the same time)

**Comment by Jiogo18** [comment]
I agreed, it would be nice to have more shortcuts available to activate PA, rather than space and the arrow keys only.

Or having two shortcuts settings would be great :
- select the character (the current setting, with space and/or arrows)
- activate the tool, just like the other tools (space/arrows by default, I'll use `;`)

For instance, press `a+;` to activate, (keep `a` pressed), then space/arrows to select 'á'.

> Therefore when PA catch the key, it shouldn't stop the action related to this key with an another program/windows feature.
> Except for a combination like letter+space or letter+arrow left/right.

I use applications (games mainly) which requires pressing a letter and space (wasd+space) or an arrow (3D software).
Sometimes PA catches the shortcut, which means I can't use space and/or arrows until I released the letter.
It's less frequent with the combination `letter+arrow`, but still happens.
I could use the blacklist, but PA is just so useful in those applications.
